caldp/20220214/dev/CALDP_20220214_COS_linux_py38_rc1.yml

```diff
@@ -0,0 +1,118 @@
+# delivery_name: CALDP_20220214_COS_final
+# creation_time: Mon Feb 14 15:00:00 2022
+# conda_version: 4.10.3
+# condabuild_version: 3.18.11
+channels:
+  - https://ssb.stsci.edu/astroconda-staging
+  - https://ssb.stsci.edu/astroconda
+  - conda-forge
+  - http://ssb.stsci.edu/astroconda
+  - stsci
```

Comment:
Remove stsci

Reply:
This is not a file I created/delivered. It is also a YML never actually used in testing. I will leave this for others to comment.